src/test/rustdoc/toggle-item-contents.rs

```diff
@@ -62,8 +62,7 @@ pub struct PrivStruct {
 }
 
 // @has 'toggle_item_contents/enum.Enum.html'
-// @count - '//details[@class="rustdoc-toggle type-contents-toggle"]' 1
-// @has - '//details[@class="rustdoc-toggle type-contents-toggle"]' 'Show fields'
+// @!has - '//details[@class="rustdoc-toggle type-contents-toggle"]'
```

Comment:
Can you also check the "DOM path" please? `!has` doesn't cover changes, only that something doesn't exist after all.

Reply:
Wouldn't this test be more general than checking a specific DOM path? I feel a bit confused about what you'd like me to change the test to. Could you give an example of what you mean?

Reply:
I was thinking that the `<details>` tag was wrapping another tag, but this is not the case. I really don't like that but I guess we can't do better.